actions/kpops-runner/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: "Kpops runner"
+description: "Run Kpops command line with the given params"
+
+inputs:
+  mode:
+    description: "The command mode you want to use to launch kpops"
+    required: true
+  working-directory:
+    description: "The root directory containing the config.yaml, pipelines folder and defaults"
+    required: true
+    default: "."
+  pipeline:
+    description: "Pipeline file you want to run"
+    required: true
+  execute:
+    description: "Set execution flag"
+    required: true
+    default: "false"
+  defaults:
+    description: "Defaults folder path"
+    required: true
+    default: "defaults"
+  config:
+    description: "Config.yaml file path"
+    required: true
+    default: "config.yaml"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check out repository code
+      uses: actions/checkout@v3
+    - run: pip install kpops==0.2.0.dev20221122085523 --extra-index-url https://test.pypi.org/simple/
```

Comment:
You can use just one step for all bash commands, see for example: https://github.com/bakdata/ci-templates/blob/main/actions/helm-gke-deploy/action.yaml#L51

Reply:
I splitted them in logical steps. Install, Dry-Run and Process :)